proto/method_string.go

```diff
@@ -9,7 +9,7 @@ const _Method_name = "GetPutConditionalPutIncrementDeleteDeleteRangeScanEndTrans
 var _Method_index = [...]uint16{0, 3, 6, 20, 29, 35, 46, 50, 64, 73, 86, 100, 105, 115, 125, 144, 164, 174, 189, 210, 236, 249, 268, 287, 300}
 
 func (i Method) String() string {
-	if i < 0 || i >= Method(len(_Method_index)-1) {
+	if i < 0 || i+1 >= Method(len(_Method_index)) {
```

Comment:
this looks spurious, are you using the latest stringer? `glock sync github.com/cockroachdb/cockroach`